test/integration/smoke/test_service_offerings.py

```diff
@@ -811,8 +815,27 @@ def test_06_disk_offering_strictness_false(self):
             "Check service offering of the VM"
         )
 
+        self.updateVmwareSettings(True)
```

Comment:
Can you revert to the actual value (can be 'true' or 'false') set before executing this test.

Reply:
the actual value before the test started is `false`, which is what it is getting reset to.

Reply:
this is true for the tests are executed immediately in a fresh env as the default value is set to `false`. But, when the test is executed in fresh/upgraded env where the config is already modified to 'true', config value is set to 'false' after the test.

Reply:
@sureshanaparti I've addressed your comment, could you please check if the changes are fine? Thanks